convoy/api/delivery_attempts.py

```diff
@@ -10,11 +10,11 @@ class DeliveryAttempt():
     def __init__(self, config):
         self.client = Client(config)
 
-    def all(self, eventdeliveryId, query):
-        response = self.client.httpGet("/eventdeliveries/%s/deliveryattempts" %  eventdeliveryId, query)
+    def all(self, event_delivery_id, query):
+        response = self.client.http_get("/eventdeliveries/%s/delivery_attempts" %  event_delivery_id, query)
```

Comment:
Same here

Reply:
Yikes. I was doing casing evaluation and changed that erroneously :/